plugins/CoreUpdater/Diagnostic/HttpsUpdateCheck.php

```diff
@@ -31,8 +32,14 @@ public function execute()
     {
         $label = $this->translator->translate('Installation_SystemCheckUpdateHttps');
 
-        if (CoreUpdater\Controller::isUpdatingOverHttps()) {
+        if (GeneralConfig::getConfigValue('force_matomo_ssl_request') === 0) {
+            $comment = 'An info message';
+            return array(DiagnosticResult::singleResult($label, DiagnosticResult::STATUS_INFORMATIONAL, $comment));
+        } elseif (Http::isUpdatingOverHttps()) {
             return array(DiagnosticResult::singleResult($label, DiagnosticResult::STATUS_OK));
+        } else {
+            $comment = 'An error message';
+            return array(DiagnosticResult::singleResult($label, DiagnosticResult::STATUS_ERROR, $comment));
         }
 
         $comment = $this->translator->translate('Installation_SystemCheckUpdateHttpsNotSupported');
```

Comment:
I think there need to be two different diagnostic comments here
- A new info diagnostic comment to explain that HTTPS has been disabled by config.
- The existing error diagnostic comment that HTTPS is not supported.

So something like:
```suggestion
if (GeneralConfig::getConfigValue('force_matomo_ssl_request') === 0) {
	$comment = $this->translator->translate('Installation_SystemCheckUpdateHttpsDisabled');
	return array(DiagnosticResult::singleResult($label, DiagnosticResult::STATUS_INFORMATIONAL, $comment));
} elseif (Http::isUpdatingOverHttps()) {
	return array(DiagnosticResult::singleResult($label, DiagnosticResult::STATUS_OK));
} else {
	$comment = $this->translator->translate('Installation_SystemCheckUpdateHttpsNotSupported');
	return array(DiagnosticResult::singleResult($label, DiagnosticResult::STATUS_ERROR, $comment));
}
```
The new info diagnostic translation could be something like: 
```
"SystemCheckUpdateHttpsDisabled": "Matomo updates using HTTPS have been disabled using the force_matomo_ssl_request=0 config setting"
```

Reply:
@peterhashair, just a small change needed with the diagnostic comments and this should be good to go :+1: